docs/source/instrument/_install_new_instrument.rst

```diff
@@ -291,20 +295,94 @@ option:
+While this step is optional, it is **highly recommended** that you keep
 your bluesky instrument directory under some form of software version
 control. At minimum, this can provide some form of backup protection. It
 also helps others to collaborate with similar bluesky instruments by
 sharing your instrument's implementations.
 
-Instructions for using `git <https://git-scm.com/>`__ as software
-version control with `GitHub <https://github.com/>`__ or the `APS GitLab
-server <https://git.aps.anl.gov/>`__ are provided in `this separate
-document <https://bcda-aps.github.io/bluesky_training/reference/_github_create_repo.html>`__.
+The installer program initializes a local git repository in the ``bluesky``
+folder. We now need to create a blank remote repository, for example
+on `GitHub <https://github.com/>`_ or the `APS GitLab server
+<https://git.aps.anl.gov/>`_ using your `beamline organization 
+<https://bcda-aps.github.io/bluesky_training/reference/_git-help.html#beamline-github-organizations>`_. 
+Note that the bluesky team recommends GitHub (why is that? See 
+`here <https://bcda-aps.github.io/bluesky_training/reference/_git-help.html>`_ 
+for more info).
+
+Note: APS beamlines uses conventions for `beamline organization 
+<https://bcda-aps.github.io/bluesky_training/reference/_git-help.html#beamline-github-organizations>`_
+
+**WARNING**: to simplify the process (avoid merge conflicts), it is important for the
+remote repository to be **empty**. To do so, carefully follow the instructions described 
+in the tabs below:
+
+
+.. tabs::
+
+   .. tab:: GitHub (recommended)
+
+      Follow the official GitHub instructions `Create a repository <https://docs.github.com/en/get-started/quickstart/create-a-repo#create-a-repository>`_
+      with the modifications below:
+
+      .. raw:: html
+
+         <ul>
+         <li><b>Do not</b> select "Initialize this repository with a README" (skip step 5)</li>
+         <li>Keep the other options as default:</li>
+         <ul>
+         <li>Visibility: Public</li>
+         <li>Repository template: No template</li>
+         <li><code>.gitignore</code> template: None</li>
+         <li>License: None</li>
+         </ul>
+         </ul>
+
+
+
+   .. tab:: GitLab
 
+      Follow the official GitLab instructions `Create a blank project <https://docs.gitlab.com/ee/user/project/#create-a-blank-project>`_
+      with the modifications below:
+
+      .. raw:: html
+
+         <ul>
+         <li>Visibility Level: Public</li>
+         <li><b>Unselect</b> "Initialize repository with a README"</li>
+         <li>Keep "Enable Static Application Security Testing (SAST)" unselected</li>
+         <li>
+         </ul>
+
+
+
+
+The next steps are common to both web-based repositories (GitHub and GitLab):
+copy the remote `repository URL <https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#choosing-a-url-for-your-remote-repository>`_, for example,
```

Comment:
insert a blank line here?

Reply:
I finds it weird since there is just one sentence. One solution would be a bullet list but I am getting this error when I build it: 

`/home/RODOLAKIS/src/bluesky_training/docs/source/instrument/_install_new_instrument.rst:354: WARNING: Bullet list ends without a blank line; unexpected unindent.`

I tried a few things, added blank lines, but I am still getting the error and the first bullet has extract blanks before and after the URL that I cannot get rid of. Do you know what is happening?

Reply:
It's the indentation on line 354.  Needs two more blank spaces at the start so the text lines up with above.  Like this:

```py
- bullet list item
Incorrect continuation of first bullet item

- new bullet item
  with more words that go with the new bullet item
```

Reply:
What's weird is that the error message assumes line 354 is new content rather than incorrect formatting of continued text from the previous line.  So that's the wrong error message in this case.

Reply:
My suggestion is below.

Reply:
My suggestions was lost, it seems